config/deployments/integration-tests/crds/vmoperator.vmware.com_virtualmachineclassbindings.yaml

```diff
@@ -0,0 +1,67 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
@gab-satchi - I'm fine with the current form of this (having CRDs under `integration-tests/`), but would be more practical to git clone and generate them on the fly?

Reply:
We are generating it [through controller-gen](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/1322/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R256-R261) with a remote repo